### ISSUE TITLE
Playlist that ran before now returns `need more than 3 values to unpack` error

### ISSUE THREAD

**Issue by stephqwu** [plamere/SmarterPlaylists#9]
![screen shot 2019-02-28 at 8 07 26 am](https://user-images.githubusercontent.com/32605661/53580465-8bb6e900-3b30-11e9-98ba-01c0b7a21107.png)

I haven't touched the playlist program since I created it, however when I tried to run it today the GUI exposed the aforementioned error. I have no idea how long this error has gone on for as I haven't run this playlist manually since its creation, but the songs in this playlist are OLD

**Comment by jessejanderson** [comment]
Is that empty box still accurately pointing to something?

**Comment by stephqwu** [comment]
@jessejanderson You're right, it was not accurately pointing to the playlist. The name of the playlist was an emoji (✨) and the default was changed to "Your favourite coffeehouse". Once I changed it to ✨ there was an error regarding how it couldn't read unicode. Once I cleared the name and left it blank (leaving only the link and user as references) the problem was solved. 

Thank you for picking that out!! I'm going to close this issue now, however the potential exists to open a different issue regarding how a unicode name was eventually defaulted to "Your favourite coffeehouse" or app behaviour in the case of unicode characters in a playlist name

**Comment by stephqwu** [comment]
Never mind, this is still an issue. I played around with it some more and realized that even after I delete that program component the issue persists
![screen shot 2019-02-28 at 11 08 37 am](https://user-images.githubusercontent.com/32605661/53591647-579bf200-3b49-11e9-809a-5538d8dcf42e.png)

**Comment by jmonfar** [comment]
I am finding exactly the same issue. Programs that ran before, and have not been edited, now give the 'need more than 3 values to unpack' error.

With only this message it is almost impossible to debug the program.

**Comment by jmonfar** [comment]
I think that it could be related with lack of URI on some playlist parameters. It could be that before it worked with playlist name and username, now perhaps it does not work with empty URI parameter.

I have modified the program to add URI for all the included playlists, and now seems to work.
